src/tequila/quantumchemistry/orbital_optimizer.py

```diff
@@ -1,11 +1,13 @@
 import numpy
 import typing
 import copy
+import pickle
 import warnings
 from dataclasses import dataclass, field
 
 from tequila import QCircuit, ExpectationValue, minimize, TequilaWarning
-from . import QuantumChemistryBase, ParametersQC, NBodyTensor
+from . import QuantumChemistryBase, ParametersQC, NBodyTensor, Molecule
+from tequila.hamiltonian.paulis import X
```

Comment:
It's a bit dangerous regarding future code addition. Could you do the following:
here: 
```python
from tequila.hamiltonian import paulis
```

and further down in your function you can just do the following shortcut
```python
X = paulis.X
```